picard/ui/matchdetailsdialog/__init__.py

```diff
@@ -0,0 +1,379 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+# Copyright (C) 2016 Rahul Raturi
+# Copyright (C) 2018 Laurent Monin
+# Copyright (C) 2019 Philipp Wolfer
+# Copyright (C) 2020 Ray Bouchard
```

Comment:
This is a new file, so it doesn't make sense to add copyright from other authors and previous years.